.github/workflows/build_docs.yml

```diff
@@ -27,6 +28,21 @@ jobs:
           git config user.name github-actions[bot]
           git config user.email 41898282+github-actions[bot]@users.noreply.github.com
 
+      - name: Install spellcheck dependencies
```

Comment:
`cspell` provides customised image for github action e.g.,
https://github.com/FAIRmat-NFDI/pynxtools/pull/665/files#diff-ba28c8d282ca58005aabe5a8d32713d35c2710f00219765dceae9b9895ab8277